pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BrokersImpl.java

```diff
@@ -215,7 +215,13 @@ public CompletableFuture<Void> shutDownBrokerGracefully(int maxConcurrentUnloadP
         WebTarget path = adminBrokers.path("shutdown")
                 .queryParam("maxConcurrentUnloadPerSec", maxConcurrentUnloadPerSec)
                 .queryParam("forcedTerminateTopic", forcedTerminateTopic);
-        return asyncPostRequest(path, Entity.entity("", MediaType.APPLICATION_JSON));
+        CompletableFuture<Void> completableFuture = new CompletableFuture<>();
+        try {
+            sync(() -> asyncPostRequest(path, Entity.entity("", MediaType.APPLICATION_JSON)));
```

Comment:
It seems the broker side doesn't respond to this request. I think the broker side should be changed too?

<img width="1022" alt="image" src="https://github.com/apache/pulsar/assets/37220920/7f8e6c05-e4cd-4934-8d93-a9b4a0507a64">

Reply:
I mean the broker doesn't handle the shutdown request properly now. For example, you can try this command to the local standalone pulsar:

```bash
curl -X POST localhost:8080/admin/v2/brokers/shutdown
```

The expected behavior is the post request finished successfully, but the actual behavior is the post request stucks and gets a timeout error.

I think broker side may need to add a `@Suspended AsyncResponse asyncResponse` and use `asyncResponse.resume` to give client a response, like other async APIs.